client/post-editor/copy-post/accordion.jsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
It's a recent enhancement, but with #6539 you can now import directly from the Lodash root as so:

```js
import { cloneDeep, isEmpty } from 'lodash';
```